front/lib/api/assistant/conversation.ts

```diff
@@ -189,7 +192,8 @@ async function renderAgentMessage(
     type: "agent_message",
     visibility: message.visibility,
     version: message.version,
-    parentMessageId: null,
+    parentMessageId:
```

Comment:
wasn't that another PR?

Reply:
It was; will check

Reply:
Very weird, on main the line is changed so not sure why it's showing a diff here 
https://github.com/dust-tt/dust/blob/main/front/lib/api/assistant/conversation.ts

Reply:
All good